plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -485,6 +486,10 @@ export class CatalogBuilder {
       discovery,
     });
 
+    const enableRawJson = config.getOptionalBoolean(
+      'catalog.enableRawJsonResponses',
```

Comment:
Since the end result for the caller is still the same old json, maybe the parameter name is a bit confusing. But I can't think of anything better really

Reply:
yeah kinda the best I could think of :sweat_smile:, maybe something like `disableRelationsResponseCompatability` instead? that way it doesn't need as much explaining either

Reply:
Switched to `catalog.disableRelationsCompatibility` instead.